ui/src/components/SampleGrid/SampleGridTable.css

```diff
@@ -151,10 +151,6 @@
   max-width: 160px !important;
 }
 
-button[disabled] {
-  opacity: 0.2;
-}
```

Comment:
Moved to `main.css`, as this is a good thing to have, and added `pointer-events: none`.